app/javascript/common/react_config.jsx

```diff
@@ -21,6 +23,9 @@ Object.assign(ReactTableDefaults, {
   defaultFilterMethod: stringFilterMethod,
   FilterComponent: textFilter,
   NoDataComponent: customNoDataComponent,
+  noDataProps: customNoDataProps,
+  LoadingComponent: customLoadingProp,
+  loadingText: null,
```

Comment:
I don't think this prop (`loadingText`) is being used anywhere. Assuming this is a change from an earlier version of this PR, you can remove it.